tests/conftest.py

```diff
@@ -35,13 +117,13 @@ def pytest_addoption(parser):
     parser.addoption(
         "--stress-tested-api",
         action="store",
-        default=f"http://{LOCALHOST}:{NODES['1']['api_port']}",
+        default=f"http://{LOCALHOST}:{NODES[0].api_port}",
```

Comment:
The reason for indexing using '<NUMBER>` was to have a well searchable and indexable log files. Now you are putting it back as it was before.

Reply:
How does the way we store test nodes impact the logs files ?

Reply:
Done.